providers/common/der/build.info

```diff
@@ -3,11 +3,11 @@ $DER_DIGESTS_H=../include/prov/der_digests.h
 $DER_DIGESTS_GEN=der_digests_gen.c
 
 GENERATE[$DER_DIGESTS_GEN]=der_digests_gen.c.in
-DEPEND[$DER_DIGESTS_GEN]=oids_to_c.pm
+DEPEND[$DER_DIGESTS_GEN]=oids_to_c.pm NIST.asn1 DIGESTS.asn1
 
 DEPEND[${DER_DIGESTS_GEN/.c/.o}]=$DER_DIGESTS_H
 GENERATE[$DER_DIGESTS_H]=der_digests.h.in
-DEPEND[$DER_DIGESTS_H]=oids_to_c.pm
+DEPEND[$DER_DIGESTS_H]=oids_to_c.pm NIST.asn1 DIGESTS.asn1
```

Comment:
To what extent do the includ edependancies matter .. e.g this one includes "internal/der.h"

Reply:
Yes.  That is, however, found by the dependency discovery, so no need to make it explicit here